sources/web/datalab/static/datalab.js

```diff
@@ -173,15 +173,33 @@ function initializeNotebookApplication(ipy, notebook, events, dialog, utils) {
       return originalSave.apply(this, /* check_lastmodified */ [ false ]);
     }
 
+    function isSample() {
+      var path = window.location.pathname;
+      var lastSepPos = path.lastIndexOf('/');
+      return path.substring(lastSepPos-23, lastSepPos) == '/datalab/docs/notebooks';
```

Comment:
I'll add that for clarity although it isn't actually necessary in Javascript. Anything less than zero is equivalent to 0 and the test still applies then.